Examples/UIExplorer/js/ActivityIndicatorExample.js

```diff
@@ -150,19 +151,50 @@ exports.examples = [
       return <ToggleAnimatingActivityIndicator />;
     }
   },
-  {
-    title: 'Custom size',
-    render() {
-      return (
-        <ActivityIndicator
-          style={[styles.centering, {transform: [{scale: 1.5}]}]}
-          size="large"
-        />
-      );
-    }
-  },
 ];
 
+if (Platform.OS === 'android') {
```

Comment:
@janicduplessis Previously, you suggested that we erase example that uses transformation.
I was thinking that if we put

```
{
  platform: 'ios',
  ...
}
```

then it would be a bit misleading as actually transformation can be done in Android as well.

If we didn't use `platform`, then the example will still be shown in Android, along with the custom size (with number) examples.

Any input?

Should we just keep it with something like this:

```
{
   title: 'Custom size (transform scale: 1.5)',
}
```